webbpsf/opds.py

```diff
@@ -2061,10 +2088,8 @@ def move_seg_global(self, segment, xtilt=0.0, ytilt=0.0, clocking=0.0, rot_unit=
         if not delay_update:
             self.update_opd(display=display)
 
-    def move_sm_local(self, xtilt=0.0, ytilt=0.0, rot_unit='urad',
-                      xtrans=0.0, ytrans=0.0, piston=0.0, trans_unit='micron', display=False,
-                      delay_update=False):
-        """ Move the secondary mirror in pose, using segment-local control coordinates.
+    def move_sm_local(self, xtilt=0.0, ytilt=0.0, rot_unit="urad", xtrans=0.0, ytrans=0.0, piston=0.0, trans_unit="micron", display=False, delay_update=False):
+        """Move the secondary mirror in pose, using segment-local control coordinates.
```

Comment:
These two functions, move_seg_local and move_sm_local, have almost exactly the same set of arguments, so it bugs me that ruff treats them inconsistently. Not sure I care enough to think we should change any of this further, but it just puzzles me about what the tool is doing.